.github/workflows/plugin-delivery.yml

```diff
@@ -71,7 +75,8 @@ jobs:
           distrib: ${{ matrix.distrib }}
           version: ${{ inputs.version }}
           release: ${{ inputs.release }}
-          artifactory_username: ${{ secrets.artifactory_username }}
-          artifactory_password: ${{ secrets.artifactory_password }}
+          nexus_username: ${{ secrets.nexus_username }}
```

Comment:
```suggestion
          nexus_username: ${{ secrets.NEXUS_USERNAME }}
```

Reply:
does it need to be the orga secrets, since it they are already defined in the calling workflow (they are in the inputs) @sc979